packages/odyssey-react-mui/src/themes/odyssey/components.ts

```diff
@@ -46,4 +46,179 @@ export const components: ThemeOptions["components"] = {
       },
     },
   },
+  MuiButton: {
+    defaultProps: {
+      disableElevation: true,
+      disableRipple: true,
+    },
+    variants: [
+      {
+        props: { variant: "primary" },
+        style: {
+          fontWeight: 600,
+          color: "#fff",
+          borderColor: "transparent",
+          backgroundColor: "#1662dd",
+
+          "&:hover, &:focus-visible": {
+            backgroundColor: "#00297a",
+          },
+
+          "&:active": {
+            backgroundColor: "#1662dd",
+          },
+
+          "&:disabled": {
```

Comment:
The `.Mui-disabled` class is also applied in this state, but it looks like no style information is associated. For safety, we might want to extend these selectors to `&:disabled, &.Mui-disabled`? I'm not strongly opinionated on this one.